(2) Vox Populi/Balance Changes/Text/en_US/UIText.xml

```diff
@@ -408,7 +408,7 @@
 			<Text>[COLOR_POSITIVE_TEXT]Married[ENDCOLOR]</Text>
 		</Row>
 		<Row Tag="TXT_KEY_DIPLO_MAJOR_CIV_DIPLO_STATE_MARRIED_TT">
-			<Text>[COLOR_POSITIVE_TEXT]Diplomatic Marriage:[ENDCOLOR] Your [ICON_INFLUENCE] Influence does not decay, you gain +1 Delegate to the World Congress, and +15% [ICON_GREAT_PEOPLE] Great Person Rate in your [ICON_CAPITAL] Capital.</Text>
+			<Text>[COLOR_POSITIVE_TEXT]Diplomatic Marriage:[ENDCOLOR] Your [ICON_INFLUENCE] Resting Influence is increased by 75, scaling with Era, and you gain +15% [ICON_GREAT_PEOPLE] Great Person Rate in your [ICON_CAPITAL] Capital.</Text>
```

Comment:
Doing the same with this line would be harder, as it's lua, and I don't think it's desirable to see the current resting influence here (it's already shown in a nearby UI anyway). Here we would actually need to calculate the influence all over again, _in LUA_:
```
int iEra = pMajor->GetCurrentEra();
if (iEra <= 0)
{
	iEra = 1;
}
iAnchor = /*75*/ GD_INT_GET(BALANCE_MARRIAGE_RESTING_POINT_INCREASE) * iEra;
```

Also, I just realized that the resting influence is updated with each era, not locked in at the time of the marriage.  That changes my thoughts on this some.